Sources/Apollo/MultipartFormData.swift

```diff
@@ -0,0 +1,111 @@
+public class MultipartFormData {
+
+	  public static let CRLF = "\r\n"
```

Comment:
Indentation of this seems to have had some problems, could you please reindent this whole file?

Reply:
Same issue on the indentation here